docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -1,168 +1,181 @@
+### Step-by-step
+
 1. Complete the [Get Started](~/start/start.md) steps before you proceed.
 
-2. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
+1. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
 
-3. Plug in the device and make sure it is registered and there are no errors on this page provisioning the profile.
+1. Plug in the device and make sure it is registered and there are no errors on this page provisioning the profile.
 
-4. On this page, click on `+ Capability`, and add `Push Notification` and `Background Modes`. For `Background Modes`, have `Remote notifications` checked.
+1. On this page, click on `+ Capability`, and add `Push Notification` and `Background Modes`. For `Background Modes`, have `Remote notifications` checked.
 
 ## Set Up Your Backend
 
 1. Set up a new Amplify project, you can skip this step if you are using an existing one.
-```
-amplify init
-```
 
-2. Add analytics to your app and allow guests and unauthenticated users send analytics events.
-```
-amplify add analytics
-```
+    ```
+    amplify init
+    ```
 
-3. Provision the backend
-```
-amplify push
-```
+1. Add analytics to your app and allow guests and unauthenticated users send analytics events.
+
+    ```
+    amplify add analytics
+    ```
+
+1. Provision the backend
+
+    ```
+    amplify push
+    ```
 
 ## Connect to Your Backend
 
 Use the following steps to connect push notification backend services to your app.
 
-1. The `Podfile` that you configure to install the AWS Mobile SDK must contain the `AWSPinpoint` pod:
+1. Add the following pods to you `Podfile`:
 
     ```ruby
-
     target :'YOUR-APP-NAME' do
       use_frameworks!
-        pod  'AWSPinpoint'
-        pod  'AWSMobileClient'
+      pod  'AWSPinpoint'
+      pod  'AWSMobileClient'
     end
     ```
 
     Run `pod install --repo-update` before you continue.
 
-2. Open the `.xcworkspace` file.
+1. Open the `.xcworkspace` file.
 
-3. Make sure the project contains `awsconfiguration.json`. This should be generated from the previous step for setting up the backend. If you used `amplify-tools` to set up your amplify project then this file will already be added to your Xcode project bundle, if you are using the CLI directly, make sure the file is added to your Xcode project by dragging it in with `copy as needed` checked.
+1. Make sure the project contains `awsconfiguration.json`. This should be generated from the previous step for setting up the backend. If you used `amplify-tools` to set up your amplify project then this file will already be added to your Xcode project bundle, if you are using the CLI directly, make sure the file is added to your Xcode project by dragging it in with `copy as needed` checked.
```

Comment:
bundle. If

Reply:
Thanks for catching this. Your feedback made me realize mentioning Amplify Tools here out of the blue can be very confusing. I rewrote this section completely.